docs/design/sql/05-client-support.md

```diff
@@ -127,31 +126,23 @@ request {
 
 response {}
 ```
+
+The server-side resources are cleared automatically when the end of data is reached. Therefore, it is not necessary to send
+the `cancel` command to the member, if the previous `execute` or `fetch` command returned the last page, or an update count.
   
 ## 3 Limitations
 
 This section describes limitations of the current implementation
 
-### 3.1 No Backward Compatibility
-
-The protocol is not backward compatible at the moment, because the SQL feature is declared as beta. Messages could be changed 
-in an incompatible way in minor and patch versions. 
-
-In order to ensure that the user receives proper error messages in case of incompatibility, we will increment the ID of the 
-message every time it is changed in an incompatible way. The user will receive an error message about the version mismatch.
-
-When the SQL is no longer in beta, the SQL protocol will become backward/forward compatible, similarly to other Hazelcast 
-components.  
-
-### 3.2 Stickiness
+### 3.1 Stickiness
 
 There is no server-side logic to re-route client requests between members. Therefore, the protocol is sticky: `fetch` and 
 `close` commands must be sent to the same member as the prior `execute` command.
 
 We are likely to add missing routing logic in future versions. However, stickiness guarantees the optimal performance. Therefore,
 we will still employ the sticky approach on the best-effort basis even after the limitation is resolved. 
```

Comment:
I think the main reason for requiring using the same member today is that the query fails anyway if the initiator is down. There's no point in the client sending `fetch` to another member if it cannot contact the initiator.

Reply:
No, the reason is as explained in the doc - we simply do not have the code to re-route client requests between members. 
We may implement some advanced routing if needed, but there is no clear indication that any use case would benefit from it, therefore we keep things simple.

Reply:
Yeah, it's what I meant - there's no benefit in re-routing the client commands. But the text suggests it's a missing feature, something that we should do.

Reply:
Ok, I removed the paragraph altogether - this is not a limitation at all, so no need to mention it.